docs/troubleshooting.md

```diff
@@ -0,0 +1,12 @@
+# Tests fail on 'Suite Setup' - 'To Boot Directly' not found
```

Comment:
`#` should happen only once in document, so use it as a name of the document, and use `##` for sections.